lib/normalize.js

```diff
@@ -1,9 +1,25 @@
 'use strict';
 
 var path = require('path');
+var urlRegex = require('url-regex');
+var normalizeUrl = require('normalize-url');
+var removeTrailingSep = require('remove-trailing-separator');
 
 function normalize(str) {
-  return str === '' ? str : path.normalize(str);
+  if (str === '') {
+    return str;
+  }
+
+  if (urlRegex().test(str) && !/^www\./.test(str)) {
```

Comment:
`url-regex` matches on strings like *www.google.de* which are also a valid directory.
So let `path.normalize` do the normalization